Dockerfile

```diff
@@ -16,22 +16,14 @@ RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key
 # Fail the build if the version doesn't match what we expected
 RUN google-chrome --version | grep " $EXPECTED_CHROME_VERSION\."
 
-# Need to analyze and format since dart_build_image only does it automatically for
-# packages that depend on dart_dev
-RUN dartanalyzer .
+# TODO: Remove this and instead run it within the github actions CI on the stable channel once SDK lower bound is >=2.9.3
 RUN dartfmt --line-length=120 --dry-run --set-exit-if-changed .
 
-RUN pub run dependency_validator -i build_runner,build_test,build_web_compilers
-
-# TODO run tests using dart_unit_test_image in skynet, remove Chrome install
+# TODO: Remove these test runs once SDK lower bound is >=2.9.3
 RUN pub run build_runner test --release -- --preset dart2js --exclude-tags=dart-2-7-dart2js-variadic-issues
 RUN pub run build_runner test -- --preset dartdevc
 
-RUN dart ./tool/run_consumer_tests.dart --orgName Workiva --repoName over_react --testCmd "pub run dart_dev test -P dartdevc"
```

Comment:
I didn't see that this was moved elsewhere, though I might have missed it. Are we not running consumer tests as part of CI anymore?

Reply:
I'd like to have them run using rmconsole. I haven't figured out why rosie isn't producing her comment to make it easier to kick off a consumer test - but they can be manually kicked off through the events interface for the time-being.

Reply:
Rosie comments aren't posted on OSS repos.

To see the comment, you can click the Rosie "Details" link at the bottom of the page and then click on the "Evaluation" tab.